html-forms/html-forms/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Html-Forms</title>
+  </head>
+  <body>
+<label for="Email">
+ <div>Email</div>
+    <input id="Email" type="text" name="Email">
+</label>
+<label for="Questions">
+ <div>Why is coding cool?</div>
+<select name="Questions">
+  <option value= "Select an option">Select an option</option>
+  <option value="Because it's intersting">Because it's intersting</option>
+  <option value="Because it's fun">Because it's fun</option>
+  <option value="Because it's powerful">Because it's powerful</option>
+</select>
+  </label>
+<label for="Favorite">
+<div>Favorite Language (choose one)
+    <br />
+<input type="radio" name="Favorite" value="HTML"
+checked="checked" /> HTML
+<input type="radio" name="Favorite" value="CSS" />
+  CSS
+  <input type="radio" name="Favorite" value="JavaScript" />
+   JavaScript
+</div>
+</label>
```

Comment:
We want to avoid using `<br>` tags, it is better to wrap things in a `<div>`. For example:

```HTML
<label for="Favorite">
  <div>Favorite Language (choose one)
    <div>
      <input type="radio" name="Favorite" value="HTML" checked="checked" />  HTML
      <input type="radio" name="Favorite" value="CSS" /> CSS
      <input type="radio" name="Favorite" value="JavaScript" /> JavaScript
    </div>
  </div>
</label>
```